docs/topics/mtr-using-openrewrite-recipes.adoc

```diff
@@ -0,0 +1,59 @@
+
+See link:{KBArticleTechnologyPreview}[Technology Preview features support scope] on the Red&nbsp;Hat Customer Portal for information about the support scope for Technology Preview features.
+====
+
+You can refactor the source code of Java applications by using link:https://docs.openrewrite.org/[OpenRewrite] recipes with the {ProductShortName} CLI.
+
+For example, the OpenRewrite recipe `org.jboss.windup.JavaxToJakarta` renames imported `javax` packages to their `jakarta` equivalents.
+
+.Procedure
+
+. Run `windup-cli`, specifying the recipe name, the path to the configuration file, and the application:
++
+[source,terminal,subs="attributes+"]
+----
+$ ./windup-cli --openrewrite --input </path/to/source/project> \
+  "-Drewrite.configLocation=<path/to/rewrite.yaml>"  \
+  "-DactiveRecipes=<recipe_name>" --goal dryRun
+----
+
+* `"-DactiveRecipes=<recipe name>"`: Specify the OpenRewrite recipe, for example, `org.jboss.windup.JavaxToJakarta`.
+
+* `--input`: Specify the application to be refactored.  The application must be the top of the source code project containing a Maven Project Object Model (POM) XML file, `pom.xml`.
+
+* `-Drewrite.configLocation=<path/to/rewrite.yaml>` : The location of the `rewrite.yaml` configuration file to use.
+ The shipped `rewrite.yaml` configuration files are located in your
+`<{ProductShortName}_HOME>/rules/openrewrite` subfolder, for example,`" -Drewrite.configLocation=<{ProductShortName}_HOME>/rules/openrewrite/jakarta/javax/imports/rewrite.yaml"`.
+
+* `"-DactiveRecipes=<recipe name>"`: Specify the OpenRewrite recipe, for example, `org.jboss.windup.JavaxToJakarta`.
++
+You can include more than one recipe by specifying each in the `activeRecipes` parameter. For example, to include the recipes `org.jboss.windup.JavaxInjectToJakartaInject` and `org.jboss.windup.JavaxEjbToJakartaEjb"`, enter the following for `"-DactiveRecipes=<recipe name>"`:
++
+[source, terminal,subs="attributes+"]
+----
+  "-DactiveRecipes=org.jboss.windup.JavaxInjectToJakartaInject, \
+    org.jboss.windup.JavaxEjbToJakartaEjb"
+----
+
+* `--goal`: Optional: The OpenRewrite Maven goal to run.
+** `dryRun` : The script returns a list of proposed changes. Ignore the `"Run 'mvn rewrite:run' to apply the recipes"` message.
+** `run`: The script applies the changes.
+
+. Run `windup-cli` with `--goal run` to apply the recipe:
++
+[source,terminal,subs="attributes+"]
+----
+$ ./windup-cli --openrewrite --input </path/to/source/project> \
+  "-Drewrite.configLocation=<path/to/rewrite.yaml>"  \
+  "-DactiveRecipes=<recipe_name>" --goal run
+----
```

Comment:
This whole section should be removed, the transformation command works differently in mta-cli and its been already documented in https://windup.github.io/windup-documentation/docs/cli-guide/master/index.html#mta-cli-transform_cli-guide

Reply:
this is to fix the MTR differences to MTA, which is why the name was changed to `mtr-using-openrewrite-recipes.adoc` and features only in the MTR master.adoc:

```
windup-documentation/docs/cli-guide-mtr/master.adoc:
   35  
   36  // Use OpenRewrite recipes
   37: include::topics/mtr-using-openrewrite-recipes.adoc[leveloffset=+3]
```